packages/commons/src/events/ActionDocument.ts

```diff
@@ -193,6 +199,9 @@ export const ActionDocument = z
     ApprovedCorrectionAction.openapi({ ref: 'ApprovedCorrectionAction' }),
     RejectedCorrectionAction.openapi({ ref: 'RejectedCorrectionAction' }),
     UnassignedAction.openapi({ ref: 'UnassignedAction' }),
+    ReviewCorrectionRequestAction.openapi({
```

Comment:
Do we want to add this to ActionDocument actually? 

https://github.com/opencrvs/opencrvs-core/issues/9301
> REVIEW_CORRECTION_REQUEST (Only UI, so never stored to event "actions" list)